package.json

```diff
@@ -69,7 +71,9 @@
     "pretest": "yarn run compile && yarn run lint",
     "test": "node ./out/test/runTest.js",
     "vsce:package": "vsce package",
-    "vsce:publish": "vsce publish"
+    "vsce:publish": "vsce publish",
+    "run-in-browser": "vscode-test-web --browserType=chromium --open-devtools --extensionDevelopmentPath=. .",
+    "dev:web": "concurrently \"yarn watch-web\" \"pnpm run-in-browser\""
```

Comment:
I would prefer only adding a single new command: `run-in-browser` to make it possible to test the extension in web-mode.

I don't really see a reason to add support for developing the extension in web-mode.

This would also eliminate the `concurrently` dependency.